### ISSUE TITLE
 libphonenumber-for-php/src/libphonenumber/PhoneNumber.php Line 554 issue

### ISSUE THREAD

**Issue by tnory56** [giggsey/libphonenumber-for-php#108]
While going over some of the code I noticed that the countryCodeSource was null but was still being sent out when the __toString method was being called.
I think that this needs to be changed from:

```php
        if ($this->hasCountryCode()) {
            $outputString .= ' Country Code Source: ' . $this->countryCodeSource;
        }
```

To:
```php
        if ($this->hasCountryCodeSource()) {
            $outputString .= ' Country Code Source: ' . $this->countryCodeSource;
        }
```

**Comment by giggsey** [comment]
Good point.

If you want, submit a Pull Request and I'll accept. Otherwise I'll make the change over the weekend.

**Comment by giggsey** [comment]
Closed by #109
